settings/common.py

```diff
@@ -333,3 +337,26 @@
 EKS_NODEGROUP_ROLE_ARN = os.environ.get("EKS_NODEGROUP_ROLE_ARN")
 
 ENVIRONMENT = os.environ.get("ENVIRONMENT", "dev")
+
+SIMPLE_JWT = {
+    "ACCESS_TOKEN_LIFETIME": timedelta(days=365),
+    "REFRESH_TOKEN_LIFETIME": timedelta(days=365),
+    "ROTATE_REFRESH_TOKENS": False,
+    "UPDATE_LAST_LOGIN": False,
+
+    "ALGORITHM": "HS256",
+    "SIGNING_KEY": "test-key",
```

Comment:
I think keeping the signing key as `test-key` is way too simple. Moreover, it should be passed as an environment variable.

Reply:
Actually we can remove this, by default it will use `SECRET_KEY` from existing django settings.